### DISCUSSION TITLE
How to add a fixation cross between each stimulus in Psychopy?

### DISCUSSION THREAD

**Discussion by Ricardo-Piccolo** [jspsych/jsPsych#3040] [General]
Hey y'all,

I'm still learning about the basics on Psychopy, but I'm having major troubles when it comes to adding a fixation cross (+) between each randomized stimulus.

I have no idea how to do it.

Can you guys help me?

**Comment by jodeleeuw** [comment]
Hey @Ricardo-Piccolo,

If you are looking for PsychoPy help you should try their support forum:
https://discourse.psychopy.org/
